#----------------------
# Image info
#----------------------
int32 WhiteFieldsNum
ColorPoint[] CheckerFields
Point MaxCorner
Point MinCorner
